### PR TITLE
APIGOV-18374 - add client timeouts for central, apimanager, apigateway, & output traceability

### PR DESCRIPTION
**Pull request by kflan-io**

### CODE REVIEW

File: pkg/api/client.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ const (
 	PUT    string = http.MethodPut
 	DELETE string = http.MethodDelete
 
-	defaultTimeout     = time.Second * 60
+	defaultTimeout     = time.Second * 30
```
Comment (jcollins-axway) [inline pkg/api/client.go:26]: Was this meant to change?

File: pkg/api/client.go [hunk 2]
```diff
@@ -78,13 +83,13 @@ func NewClient(cfg config.TLSConfig, proxyURL string) Client {
 }
 
 func getTimeoutFromEnvironment() time.Duration {
-	cfgHTTPClientTimeout := os.Getenv("HTTP_CLIENT_TIMEOUT")
+	cfgHTTPClientTimeout := os.Getenv("CENTRAL_CLIENTTIMEOUT")
```
Comment (jcollins-axway) [inline pkg/api/client.go:86]: Was this supposed to change? CENTRAL_CLIENTTIMEOUT is for central specifically to override what the HTTP_CLIENT_TIMEOUT might have

File: pkg/api/client.go (hunk 1 above)
Comment (kflan-io) [reply pkg/api/client.go:26]: Yes; the story asks for 30 second defaults. I contemplated adding a new variable.

Comment (jcollins-axway) [reply pkg/api/client.go:26]: This is good then.

File: pkg/api/client.go (hunk 2 above)
Comment (sbolosan) [reply pkg/api/client.go:86]: following...

File: pkg/api/client.go (hunk 1 above)
Comment (sbolosan) [reply pkg/api/client.go:26]: sorry, but did C know that it was at 60 seconds?  I know the story said 30, but I think we moved it to 60 to accommodate the huge petstore swagger suman was testing (if i remember correctlY)

File: pkg/api/client.go (hunk 2 above)
Comment (kflan-io) [reply pkg/api/client.go]: resolved

**Review by jcollins-axway — APPROVED** [review]

**Review by sbolosan — APPROVED** [review]

**Pull request merged**